### ISSUE TITLE
An error occured while adding the project

### ISSUE THREAD

**Issue by deadcoder0904** [Kruemelkatze/vscode-dashboard#29]
I'm basically trying to add a project. It worked last night. Today I updated VSCode.

I think this is because of the new update. My VSCode version -

```
Version: 1.40.0
Commit: 86405ea23e3937316009fc27c9361deee66ffbf5
Date: 2019-11-06T17:09:34.601Z
Electron: 6.1.2
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Darwin x64 19.0.0
```

**Comment by felipefa** [comment]
Same error here.

My VSCode version:
```
Version: 1.40.0
Commit: 86405ea23e3937316009fc27c9361deee66ffbf5
Date: 2019-11-06T18:14:08.920Z
Electron: 6.1.2
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Linux x64 4.15.0-66-generic
```

**Comment by Kruemelkatze** [comment]
I'll check this. Thanks for reporting!

**Comment by Kruemelkatze** [comment]
I just published ***1.5.4*** which fixes this.

It's either a bug in vscode 1.40.0, a fixed bug that the dashboard happened to use, or an undocumented breaking change in the API. [We will see](https://github.com/microsoft/vscode/issues/84325)

I noticed that all edit icons on the dashboard are shown upon opening on vscode 1.40.0. I'll fix this soon, as that's not that urgent. :)

**Comment by deadcoder0904** [comment]
Thank you so much, you're really fast at fixing stuff.

I don't know why it required me to "Reload Extension" but it works now :)
